src/reference-manual/blocks.Rmd

```diff
@@ -565,4 +565,13 @@ After the generated quantities statements are executed, the constraints
 on the declared generated quantity variables are validated.
 
 All variables declared as generated quantities are printed as part of
```

Comment:
Technically, what I wrote is correct.  Variables declared in nested blocks are local variables, not generated quantities.  Only the first variables declared at the top are considered generated quantities.  

I would rather rephrase this to add another sentnece that says that variables declared in local scopes are local variables and not generated quantities and thus won't be printed.  Same for transformed parameters and transformed data---in general, local variables are not visible beyond the block in which they're declared, which means they're not visible for output.
But I'll just merge this if it was confusing before as nobody's going to get confused by the semantic implication that